python/smqtk/web/search_app/modules/iqr/iqr_search.py

```diff
@@ -315,45 +299,124 @@ def iqr_session_state():
             descriptor vectors.
             """
             with self.get_current_iqr_session() as iqrs:
-                iqrs_uuid = str(iqrs.uuid)
-                pos_elements = list(set(
-                    # Pos user examples
-                    [tuple(d.vector().tolist()) for d
-                     in self._iqr_example_pos_descr[iqrs.uuid].values()] +
-                    # Adjudicated examples
-                    [tuple(d.vector().tolist()) for d
-                     in iqrs.positive_descriptors],
-                ))
-                neg_elements = list(set(
-                    # No negative user example support yet
-                    # Adjudicated examples
-                    [tuple(d.vector().tolist()) for d
-                     in iqrs.negative_descriptors],
-                ))
-
-            z_buffer = BytesIO()
-            z = zipfile.ZipFile(z_buffer, 'w', zipfile.ZIP_DEFLATED)
-            z.writestr(iqrs_uuid, json.dumps({
-                'pos': pos_elements,
-                'neg': neg_elements,
-            }))
-            z.close()
-
-            z_buffer.seek(0)
-
-            return flask.send_file(
-                z_buffer,
+                 print("external_pos1:",self._iqr_example_pos_descr[iqrs.uuid])
+                 state_b64=iqrs.get_state_bytes(self._iqr_example_pos_descr[iqrs.uuid])
+                 #print("state_b6i4",state_b64)
+                 state_bytes=state_b64
+                 #state_bytes = base64.b64(state_b64).decode('utf-8')
+                 state_dict = json.load(
+                zipfile.ZipFile(
+                    BytesIO(state_bytes),
+                    'r',
+                    IqrSession.STATE_ZIP_COMPRESSION
+                ).open(IqrSession.STATE_ZIP_FILENAME)
+            )
+                 working_data = {}                 
+                 sid_data_elems = self._iqr_example_data.get(iqrs, {})
+                 for uid, workingElem in six.iteritems(sid_data_elems):
+                    working_data[uid] = {
+                    'content_type': workingElem.content_type(),
+                    'bytes_base64':
+                        base64.b64encode(workingElem.get_bytes()),
+                }
+                 print("working_data",working_data)
```

Comment:
Probably want to remove this print so the log is not bloated with huge base64 strings.